packages/ui-react/src/components/FormFeedback/FormFeedback.module.scss

```diff
@@ -30,3 +30,14 @@
   color: theme.$form-feedback-success-color;
   background-color: theme.$form-feedback-success-bg-color;
 }
+
+.hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+}
```

Comment:
Maybe we can move this to `accessibility.scss`, we might use it elsewhere as well for the same purpose

Reply:
I'm also in favour of doing that. Please be sure to test is afterwards ;)

Reply:
Tested, zipped and shipped.